cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillController.java

```diff
@@ -543,7 +556,9 @@ private CompletionStage<String> deleteJobAndGetCompletionStatus(int gracePeriodS
     CompletableFuture<String> resultFuture = new CompletableFuture<>();
     String name = meta.getName();
     if (jobStatus == null) {
-      isStopped = true;
+      if (!jobTimedOut) {
```

Comment:
Shall we add a log statement if job times out?

Reply:
added log